### ISSUE TITLE
NSFW images do not get downloaded

### ISSUE THREAD

**Issue by rexmightlag** [husakki/scrolller_scraper#5]
When trying to download from NSFW image boards, the script returns:

```
Scrolling to load images
Finding elements that contain the image...
Found: 0
```

over and over again

**Comment by husakki** [comment]
Yes, I'm aware of the issue and it will be tackled with the other issues as soon as I get some other projects sorted out.
The problem is that, when you first open a nsfw page you will have to confirm that your 18, and this step needs to be checked by the script.
